src/Storage/DatabaseStorage.php

```diff
@@ -124,15 +124,16 @@ public function store(Collection $items): void
     public function trim(): void
     {
         $now = CarbonImmutable::now();
+        $keep = $this->config->get('pulse.ingest.trim.keep');
```

Comment:
We probably need to introduce a custom storage key here: `pulse.storage.keep`

Then we need to decide if we fallback to the ingest value if the storage value is not specified.

Reply:
Hi @timacdonald, we thought about that because the pulse.ingest.trim.lottery key is already used in the case of DatabaseStorage, so why not use the pulse.ingest.trim.keep too, to customize if we want to keep the data longer ? Or as you said, create a new key in the storage config part.

https://github.com/laravel/pulse/blob/419a9bfd3b7c537db9e086f143c2b828693fe9e7/src/Pulse.php#L310